### PR TITLE
stream: add communities stream

### PR DESCRIPTION
**Pull request by zzacharo**

### CODE REVIEW

File: site/zenodo_rdm/migrator/streams.yaml [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
   extract:
-    filepath: /path/to/users.jsonl
+    filepath: /path/to/communities.jsonl
   load:
     db_uri: postgresql://zenodo:zenodo@localhost:5432/zenodo
     tmp_dir: /path/to/tmp
+records:
+  extract:
+    filepath: /path/to/records.jsonl
+  load:
```
Comment (kpsherva) [inline site/zenodo_rdm/migrator/streams.yaml:16]: you said you want to change this :)

File: site/zenodo_rdm/migrator/records.jsonl [hunk 2]
```diff
@@ -0,0 +1,10 @@
...type": {"type": "dataset"}}, {"scheme": "doi", "relation": "hasPart", "identifier": "10.5281/zenodo.5684777", "resource_type": {"type": "publication", "subtype": "taxonomictreatment"}}, {"scheme": "url", "relation": "hasPart", "identifier": "http://treatment.plazi.org/id/039A87860F7EDE1DFF4C0E643C77FE6E", "resource_type": {"type": "publication", "subtype": "taxonomictreatment"}}, {"scheme": "doi", "relation": "hasPart", "identifier": "10.5281/zenodo.206056", "resource_type": {"type": "image", "subtype": "figure"}}, {"scheme": "doi", "relation": "hasPart", "identifier": "10.5281/zenodo.206057", "resource_type": {"type": "image", "subtype": "figure"}}, {"scheme": "doi", "relation": "hasPart", "identifier": "10.5281/zenodo.206058", "resource_type": {"type": "image", "subtype": "figure"}}, {"scheme": "url", "relation": "hasPart", "identifier": "http://table.plazi.org/id/DF4C66180F7ADE1AFF4C09973CB3FF34", "resource_type": {"type": "dataset"}}], "alternate_identifiers": [{"scheme": "lsid", "identifier": "urn:lsid:plazi.org:pub:FFA3FFFE0F7EDE1EFFDB090F392FFF9A"}, {"scheme": "url", "identifier": "http://publication.plazi.org/id/FFA3FFFE0F7EDE1EFFDB090F392FFF9A"}]},"version_id":6,"index":0}
```
Comment (slint) [inline site/zenodo_rdm/migrator/records.jsonl:1]: Is this example file needed here?

File: site/zenodo_rdm/migrator/transform.py [hunk 3]
```diff
@@ -1,10 +1,113 @@
...
+        """Returns the creation date of the record."""
+        return entry["created"]
+
+    def _updated(self, entry):
+        """Returns the creation date of the record."""
+        return entry["updated"]
+
+    def _version_id(self, entry):
+        """Returns the version id of the record."""
+        # TODO: use ranking?
```
Comment (slint) [inline site/zenodo_rdm/migrator/transform.py:71]:
```suggestion
```

**Review by slint — APPROVED** [review]
LGTM! Thanks a lot for the loader :)

File: site/zenodo_rdm/migrator/records.jsonl (hunk 2 above)
Comment (zzacharo) [reply site/zenodo_rdm/migrator/records.jsonl]: Deleted it :)

**Pull request merged**